### PR TITLE
feat(wip): removed signer from viewMethods by dropping NAJ account.ViewFunction

### PR DESCRIPTION
**Pull request by jaswinder6991**
In this PR, I remove account.ViewFunction to drop dependency on using a Signer for view methods. We instead use the providers from near-api-js to write a utility method for viewFunction.

# Description

# Type of change

- [x ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🏗️ Build configuration (CI configuration, scaffolding etc.)
- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [x ] 📝 Documentation update(s)
- [ ] 📦 Dependency update(s)
- [ ] 👩🏽‍💻 Improve developer experience
- [ ] ⚡ Improve performance
- [x ] ✨ New feature (non-breaking change which adds functionality)
- [x ] ♻ Refactor
- [ ] ⏪ Revert changes
- [ x] 🧪 New tests or updates to existing tests

### CODE REVIEW

File: src/constants/Config.ts
Comment (kieranroneill) [inline src/constants/Config.ts:1]: This should be moved to the `/types` directory.

File: src/controllers/Social.ts
Comment (kieranroneill) [inline src/controllers/Social.ts:121]: To conform to the other function naming, private methods should be prefixed with an underscore: `_isStorageBalance`

File: src/utils/rpcQueries/viewFunction.ts
Comment (kieranroneill) [inline src/utils/rpcQueries/viewFunction.ts:33]: Like above, this will fail without a polyfill on browsers. I recommed we use a library to handle base64: https://www.npmjs.com/package/@stablelib/base64

File: src/utils/rpcQueries/viewFunction.ts
Comment (kieranroneill) [inline src/utils/rpcQueries/viewFunction.ts:17]: `Buffer` is not a native object in browsers. You need to use either:
* a polyfill - https://github.com/feross/buffer
* or [`TextDecoder`](https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder) as it is in both the browser and Node.js (https://nodejs.org/api/util.html#class-utiltextdecoder)
* or use a library like: https://www.npmjs.com/package/@stablelib/utf8

**Review by kieranroneill — CHANGES_REQUESTED** [review]
Some minor conventions that are not gamechangers. However, the use of `Buffer` in the `viewFunction` will not work in browsers without a polyfill.

I recommendation would be to use individual packages (base64/utf8) from https://www.stablelib.com/index.html to do all the decoding/encoding.

File: src/utils/rpcQueries/viewFunction.ts
Comment (kieranroneill) [inline src/utils/rpcQueries/viewFunction.ts:33]: This will fail in Node.js context as `btoa` is only in the browser. 

Suggestion: you could wrap it in a try/catch and fallback to `Buffer.toString('base64)`.

**Review by kieranroneill — CHANGES_REQUESTED** [review]

File: src/utils/rpcQueries/viewFunction.ts
Comment (kieranroneill) [inline src/utils/rpcQueries/viewFunction.ts:20]: 👍

**Review by kieranroneill — APPROVED** [review]
Approved ✅

**Comment by jaswinder6991** [comment]
@race-of-sloths

**Comment by race-of-sloths** [comment]
@jaswinder6991 Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
New Sloth joined the Race! Welcome!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/jaswinder6991?type=bot&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/jaswinder6991?type=bot&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/jaswinder6991)


<details><summary>Current status: <i>waiting for scoring</i></summary>

We're waiting for maintainer to score this pull request with `@race-of-sloths score [0,1,2,3,5,8,13]` command
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/jaswinder6991)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Comment by race-of-sloths** [comment]
### ✅ PR is finalized! 
Your contribution is much appreciated with a final score of 0!
You have received 10 (0 base + 10 weekly bonus) Sloth points for this contribution


Another weekly streak completed, well done @jaswinder6991! To keep your weekly streak and get another bonus make pull request next week! Looking forward to see you in race-of-sloths

**Pull request merged**
